delete-user-data/extension.yaml

```diff
@@ -18,8 +18,8 @@ specVersion: v1beta
 version: 0.1.2
 
 description:
-  Deletes data keyed on a userId from Cloud Firestore, Realtime
-  Database, and/or Cloud Storage when a user deletes their account.
```

Comment:
jFYI - the console and the CLI do not respect line breaks in these parameter descriptions.  Given this, I wouldn't spend a lot of time adjusting these line breaks in the extension.yaml file, unless it's just for readability in the extension.yaml file (which is, of course, a nice-to-have).

Reply:
Looks like these were changed because of formatting (the line width rule). I can change back but if we introduce CI checks, this may need to be reverted to be linted correctly.

Reply:
naw, I'd just leave the changes that are already in this PR.

If we do eventually get control of how these descriptions line break in the console and CLI, we'll need to be sure that any checks here in GitHub play nice :-)